pkg/kubelet/dockershim/cm/BUILD

```diff
@@ -41,6 +42,7 @@ go_library(
             "//pkg/kubelet/dockershim/libdocker:go_default_library",
             "//staging/src/k8s.io/apimachinery/pkg/util/version:go_default_library",
             "//staging/src/k8s.io/apimachinery/pkg/util/wait:go_default_library",
+            "//vendor/github.com/opencontainers/runc/libcontainer/cgroups:go_default_library",
```

Comment:
nit: indent

Reply:
it was generated by `hack/update-bazel.sh`